pkgs/top-level/all-packages.nix

```diff
@@ -680,6 +680,7 @@ let
 
   azureus = callPackage ../tools/networking/p2p/azureus { };
 
+  b2 = backblaze-b2;
```

Comment:
@zimbatm ouch, yes you are right. I've lost this during reassembly of my changes for the commits. Thanks for pointing out.